packages/fxa-auth-server/package.json

```diff
@@ -31,9 +31,8 @@
     "paypal-processor": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/paypal-processor.ts",
     "subscription-reminders": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/subscription-reminders.ts",
     "emails-scss": "node -r esbuild-register ./lib/senders/emails/sass-compile-files.ts",
-    "start-mjml": "NODE_ENV=dev node -r esbuild-register ./bin/mjml-server.ts",
-    "storybook": "npm run emails-scss && start-storybook -p 6010 --no-version-updates",
-    "storybook-mjml": "nodemon -e scss,ts  --exec \"grunt merge-ftl && npm-run-all -p watch-ftl start-mjml storybook\"",
+    "storybook": "npm run install-ejs && npm run emails-scss && start-storybook -p 6010 --no-version-updates -s ./",
```

Comment:
🎉 I didn't like calling it `storybook-mjml` anyway even when we had the mjml server. 😝